src/domain/RacingGame.js

```diff
@@ -0,0 +1,40 @@
+import { generateRandomNumber } from "../utils/GeneratorNumber.js";
+import Car from "./Car.js";
+
+export function createRacingCars(name) {
+  let cars = [];
+  const carNames = name.split(",");
+  carNames.forEach((name) => {
+    const car = new Car(name.trim());
+    cars.push(car);
+  });
+
+  return cars;
```

Comment:
```suggestion
  return name
    .split(",")
    .map((name) => new Car(name.trim()));
```
이렇게 표현 가능하겠네요!

만약에 Car에 of라는 static method를 만든다면

```suggestion
  return name.split(",").map(Car.of);
```
이렇게 표현할 수도 있답니다 ㅎㅎ

보통 Java에서 많이 사용하는 방식이에요!

Reply:
와 정말 코드가 심플해졌네요 
감사합니다. 참고하겠습니다 👍🏻